elasticmagic/expression.py

```diff
@@ -479,6 +479,15 @@ def _collect_doc_classes(self):
         )
 
 
+class SortScript(Expression):
+    __visit_name__ = 'sort_script'
+
+    def __init__(self, script, type_sort=None, order=None):
```

Comment:
Think `type_sort` should be renamed to `script_type`:
```suggestion
    def __init__(self, script, script_type=None, order=None):
```